content/en/docs/tutorials/security/ns-level-pss.md

```diff
@@ -80,7 +80,7 @@ The output is similar to this:
 namespace/example created
 ```
 
-## Apply Pod Security Standards
+## Enable Pod Security Standard checking for that namespace
```

Comment:
I'm not sure about using the singular `Standard` in place of `Standards` here and below. I could see using the singular when refering to a specific profile (privileged/baseline/restricted), but I think "Standards" is part of the name elsewhere... WDYT?

Reply:
I was thinking that readers would know that the singular of standards is standard. SIG Docs reviewers: thoughts?

Reply:
The first item states: `Enable Pod Security Standards on this namespace using labels...`.

Some other alternatives for a heading:
Apply Pod Security Standard checks
Enable Multiple Pod Security Standard checks
Enable Pod Security Standard checks
Use Labels to apply Pod Security Standards